src/Http/Client/FormData.php

```diff
@@ -239,7 +240,7 @@ public function contentType()
      *
      * @return string
      */
-    public function __toString()
+    public function __toString(): string
```

Comment:
It would be best to avoid adding type hint's to PHP's magic methods to be safe for future.

Reply:
What do you mean?

Reply:
I wouldn't expect PHP to change the contract on `__toString` any time soon.